frontends/web/package.json

```diff
@@ -5,6 +5,7 @@
   "devDependencies": {
     "@testing-library/jest-dom": "^5.14.1",
     "@testing-library/react": "^12.1.5",
+    "@testing-library/react-hooks": "^8.0.1",
```

Comment:
[about this (npm page)](https://www.npmjs.com/package/@testing-library/react-hooks) 🙂